src/Domain/Campaign.php

```diff
@@ -30,6 +31,13 @@ class Campaign extends SalesforceReadProxy
      */
     protected Charity $charity;
 
+    /**
+     * @ORM\ManyToMany(targetEntity="CampaignFunding", mappedBy="campaigns")
+     * @psalm-suppress PossiblyUnusedProperty Used in Doctrine ORM mapping
+     * @psalm-suppress PropertyNotSetInConstructor Set by Doctrine on inverse side too
+     */
+    protected Collection $campaignFundings;
```

Comment:
Are you sure this property needs to be here? What goes wrong if it's deleted, isn't the one on CampaignFunding enough?

If it does need to be here then I think it's better to lose the second psalm annotation and set it to empty in the constructor, so if we do put code in to use it won't crash when we try to use it with an instance that didn't come from the EntityManger.

```
$this->campaignFundings = new ArrayCollection();
```

Reply:
@bdsl Pretty sure we need it. If I remove it the query gives

`[Semantical Error] line 0, col 171 near 'availableCf WITH': Error: Class MatchBot\Domain\Campaign has no association named campaignFundings`

When we have it it seems to work end to end – though it's hard for me to be 100% certain because my Docker-ised PHP CLI segfaults on most commands at the moment. I understand this is fixed in the PHP 8.2+ images so am trying to avoid spending time on it specifically.

Added constructor initaliser on 65c5f6d8962012b41f603ab0e057d716c5696a63